apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessPreviewList.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  getQualifiedRecruitmentStatusChip,
+  isQualifiedFinalDecision,
+} from "~/utils/poolCandidate";
+
+import { RecruitmentDate } from "./MetadataDate";
+import ReviewRecruitmentProcessDialog from "./ReviewRecruitmentProcessDialog";
+
+const ReviewRecruitmentProcessPreviewList_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewRecruitmentProcessPreviewList on PoolCandidate {
+    ...ReviewRecruitmentProcessDialog
+    id
+    finalDecisionAt
+    suspendedAt
+    placedAt
+    finalDecision {
+      value
+    }
+    pool {
+      id
+      name {
+        localized
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+    }
+  }
+`);
+
+interface ReviewRecruitmentProcessPreviewListProps {
+  recruitmentProcessesQuery: FragmentType<
+    typeof ReviewRecruitmentProcessPreviewList_Fragment
+  >[];
+}
+
+const ReviewRecruitmentProcessPreviewList = ({
+  recruitmentProcessesQuery,
+}: ReviewRecruitmentProcessPreviewListProps) => {
+  const intl = useIntl();
+
+  const recruitmentProcesses = getFragment(
+    ReviewRecruitmentProcessPreviewList_Fragment,
+    recruitmentProcessesQuery,
+  ).filter(
+    (recruitmentProcess) =>
+      recruitmentProcess.finalDecisionAt &&
+      isQualifiedFinalDecision(recruitmentProcess.finalDecision?.value),
+  ); // filter for qualified recruitment processes
+
+  return (
+    <>
+      {recruitmentProcesses.length ? (
+        <PreviewList.Root>
+          {recruitmentProcesses.map((recruitmentProcess) => {
+            const { id, pool, finalDecisionAt } = recruitmentProcess;
+
+            const status = getQualifiedRecruitmentStatusChip(
+              recruitmentProcess.suspendedAt,
+              recruitmentProcess.placedAt,
+              intl,
+            );
+
+            const applicationMetadata: PreviewMetaData[] = [
+              {
+                key: "status",
+                type: "chip",
+                color: status.color,
+                children: status.label,
+              },
+              {
+                key: "classification",
+                type: "text",
+                children: pool?.classification
+                  ? getClassificationName(pool?.classification, intl)
+                  : intl.formatMessage(commonMessages.notFound),
+              },
+              {
+                key: "date",
+                type: "text",
+                children: (
+                  <RecruitmentDate
+                    finalDecisionAt={finalDecisionAt}
+                    status={status.value}
+                  />
+                ),
+              },
+            ];
+
+            return (
+              <PreviewList.Item
+                key={id}
+                title={
+                  pool.name?.localized
+                    ? intl.formatMessage(
+                        {
+                          defaultMessage:
+                            "{poolName} <hidden>recruitment process</hidden>",
```

Comment:
Space should be in hidden text: `{poolName}<hidden> recruitment process</hidden>`

Reply:
Nice catch! Fixed in [cd98a19](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/cd98a19c3e3ee182c0b81d16570dc25c194c723a)